presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -268,6 +268,21 @@ public ExecutionFailureInfo getFailureInfo()
         return warnings;
     }
 
+    public boolean isRunning()
```

Comment:
The encapsulation doesn't seem to improve things much, they read roughly the same.  I'd remove.

Reply:
It does, IMHO.
It is better to have "x.isRunning()" in many places rather than "x.getState() == State.RUNNING".
Takes less spaces, easier to read.
Also, it is not encapsulation, it is helper. :)

Reply:
There's an argument to be made for that.  Can this be done in a different PR, rather than adding it here and not updating other usages to this form?

Reply:
:)
I don't want to argue about it.
It is small thing - who wants to use that helper should and who does not - should not.
No reason of moving that small change to another diff.